app/components/review_gcse_qualification_component.html.erb

```diff
@@ -0,0 +1,24 @@
+<section class="app-summary-card govuk-!-margin-bottom-6">
```

Comment:
Very close! You can refactor this further and use the `SummaryCardComponent` by passing in `rows: gcse_qualifications_rows`. Have a look at `degrees_review_component.html.erb` as an example.

Reply:
Can we rename this component to be `GcseQualificationReviewComponent` to follow how we've named our other review components?